plugins/k8s-rebase/docs/k8s-rebase-patterns.md

```
@@ -0,0 +1,590 @@
+### Conformance suite rename (network-policy-api v0.2.0)
+
+| Old | New |
+|---|---|
+| `SupportAdminNetworkPolicy` | `SupportClusterNetworkPolicy` |
+| `SupportBaselineAdminNetworkPolicy` | `SupportClusterNetworkPolicy` (merged with ANP, dedup after) |
+| `ConformanceProfileName` type cast | `CNPConformanceProfileName` |
+
+The v0.2.0 conformance suite also expects `ClusterNetworkPolicy`
+resources in `v1alpha2` API version. If the project only installs
+`v1alpha1` CRDs, conformance tests fail with:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)